dvc/proc/manager.py

```diff
@@ -63,17 +64,30 @@ def spawn(self, args: Union[str, List[str]], name: Optional[str] = None):
             pid,
         )
 
-    def send_signal(self, name: str, signal: int):
+    def send_signal(self, name: str, sig: int):
         """Send `signal` to the specified named process."""
-        raise NotImplementedError
-
-    def kill(self, name: str):
-        """Kill the specified named process."""
-        raise NotImplementedError
+        # pylint: disable=no-member
+        process_info = self[name]
+        if os.name == "nt":
+            if sig not in (
+                signal.SIGTERM,
+                signal.CTRL_C_EVENT,  # type:ignore
```

Comment:
Here only with `sys.platform` check can prevent these mypy checking disabled here.

Reply:
If mypy prefers `sys.platform` then we can just use that